src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsGrid.css

```diff
@@ -2,127 +2,445 @@
 }
-.nb-grid-stack > .grid-stack-item .grid-stack-item-content .actionbar {
-	top: -14px;
+.nb-grid-stack.grid-stack > .grid-stack-item .grid-stack-item-content .actionbar {
 	z-index: 999;
 }
-.nb-grid-stack > .grid-stack-item.notebook-cell.active .actionbar {
-	z-index: 1;
-	transform: translate(0px, 7px);
-}
-.nb-grid-stack > .grid-stack-item.ui-draggable-dragging > .grid-stack-item-content,.nb-grid-stack > .grid-stack-item.ui-resizable-resizing > .grid-stack-item-content{
+.nb-grid-stack.grid-stack > .grid-stack-item.ui-draggable-dragging > .grid-stack-item-content,.nb-grid-stack.grid-stack > .grid-stack-item.ui-resizable-resizing > .grid-stack-item-content{
 	box-shadow:1px 4px 6px rgba(0,0,0,.2);
 	opacity:.8
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-se,.nb-grid-stack > .grid-stack-item > .ui-resizable-sw{
+
+.nb-grid-stack.grid-stack.grid-stack > .grid-stack-item > .ui-resizable-se,.nb-grid-stack.grid-stack.grid-stack > .grid-stack-item > .ui-resizable-sw {
 	background-image:url("../media/light/notebook_views_card_handle.svg");
 	background-repeat:no-repeat;
 	background-position:center;
 	transform: rotate(0deg);
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-nw{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-nw{
 	cursor:nw-resize;
 	width:20px;
 	height:20px;
 	left:10px;
 	top:0
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-n{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-n{
 	cursor:n-resize;
 	height:10px;
 	top:0;
 	left:25px;
 	right:25px
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-ne{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-ne{
 	cursor:ne-resize;
 	width:20px;
 	height:20px;
 	right:10px;
 	top:0
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-e{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-e{
 	cursor:e-resize;
 	width:10px;
 	right:10px;
 	top:15px;
 	bottom:15px
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-se{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-se{
 	cursor:se-resize;
 	width:20px;
 	height:20px;
-	right:5px;
-	bottom:0
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-s{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-s{
 	cursor:s-resize;
 	height:10px;
 	left:25px;
 	bottom:0;
 	right:25px
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-sw{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-sw{
 	cursor:sw-resize;
 	width:20px;
 	height:20px;
 	left:10px;
 	bottom:0
 }
-.nb-grid-stack > .grid-stack-item > .ui-resizable-w{
+.nb-grid-stack.grid-stack > .grid-stack-item > .ui-resizable-w{
 	cursor:w-resize;
 	width:10px;
 	left:10px;
 	top:15px;
 	bottom:15px
 }
+
+.nb-grid-stack.grid-stack text-cell-component .notebook-preview {
+	padding: 0px 0px 0px;
+}
+
+.nb-grid-stack.grid-stack output-area-component .notebook-output {
+	padding: 0px 0px 0px;
+}
+
+notebook-view-component .modal{
+	position: absolute;
+	background: rgba(244, 244, 244, .3);
```

Comment:
Colors generally shouldn't be hardcoded in css like this - does this pass a11y requirements for colors on different themes?

Reply:
I changed the colors to use existing theme colors. I also moved these styles to the modal component and its own css file.